### DISCUSSION TITLE
new blocks delay

### DISCUSSION THREAD

**Discussion by nanthandev12** [blockscout/blockscout#8597] [Support]
im running a blockchain which produces blocks only when there is new transactions and other time 1block per hour.
so if i make any transaction it takes like 5 -10 mintues to get indexed .
how can i solve this

**Comment by sl1depengwyn** [comment]
Do you have `ETHEREUM_JSONRPC_WS_URL` env var set?

**Reply by nanthandev12** [reply]
yes

**Reply by nanthandev12** [reply]
![image](https://github.com/blockscout/blockscout/assets/78531206/4ce6bb63-0c90-4371-afa7-e5e26a32dc55)
